components/List/src/MDCSelfSizingStereoCell.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#import <UIKit/UIKit.h>
+#import "MDCBaseCell.h"
+
+@interface MDCSelfSizingStereoCell : MDCBaseCell
```

Comment:
Please add class documentation describing the intended use of this cell.

Reply:
@jverkoey, in addition to describing the intention behind the cell and how to use it can the documentation refer to the example?